scripts/plan-ci.py

```
@@ -0,0 +1,820 @@
+    packages: list[dict[str, str]],
+    profile: str,
+    raw: object,
+) -> list[str]:
+    workspace_names = [package["name"] for package in packages]
+    if raw is not None:
+        affected = _string_list(raw, "affected_crates")
+    elif profile in {"main", "manual-full"}:
+        affected = workspace_names
+    else:
+        script = root / "scripts" / "affected-crates.sh"
+        result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)